### PR TITLE
chore: Trigger Argos with white-listed users

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [X] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | - |
| 🇨🇳 Chinese | - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at f843956</samp>

Add a new workflow file `.github/workflows/trigger-argos-manually.yml` to enable manual triggering of Argos visual testing by commenting "@trigger-argos" on a pull request. This allows updating the baseline or checking visual changes that are not detected automatically.

### <samp>🤖 Generated by Copilot at f843956</samp>

* Add a new workflow to trigger Argos manually by commenting "@trigger-argos" on a pull request ([link](https://github.com/ant-design/ant-design/pull/45115/files?diff=unified&w=0#diff-ed9869a5a7f59f1938ef4e5622b73d806b09ad62f4c4579ef63921ff3bb92b14R1-R25))

### CODE REVIEW

**Comment by kiner-tang** [comment]
@trigger-argos

**Comment by kiner-tang** [comment]
@trigger-argos

**Review by li-jia-nan — APPROVED** [review]

File: .github/workflows/trigger-argos-manually.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+on: [push]
+
+permissions:
+  contents: read
+
+jobs:
+  trigger_argos_with_whitelist_prs:
+    runs-on: ubuntu-latest
+    if: github.actor == 'kiner-tang'
```
Comment (yoyo837) [inline .github/workflows/trigger-argos-manually.yml:11]: 感觉是无权限专用的, yml要不要换个名字?🤔

Comment (kiner-tang) [reply .github/workflows/trigger-argos-manually.yml:11]: 恩恩，我改个名字吧

**Pull request merged**